### DISCUSSION TITLE
Multiple VTM instances

### DISCUSSION THREAD

**Discussion by demiantres** [mapsforge/vtm#888] [Q&A] [answered]
The VTM library is using static variables at multiple places in the source code. I can instantiate multiple instances of VTM but is this also recommended? Or are there known problems with running multiple instances in the same app?

**Comment by devemux86** [comment, marked as answer]
It is not supported, see https://github.com/mapsforge/vtm/issues/194.

**Reply by demiantres** [reply]
Thanks.
